zebra-consensus/src/chain.rs

```diff
@@ -79,33 +90,82 @@ where
         let mut state_service = self.state_service.clone();
         let max_checkpoint_height = self.max_checkpoint_height;
 
+        let height = block.coinbase_height();
+        // Report each 1000th block at info level
+        let info_log = matches!(height, Some(BlockHeight(height)) if (height % 1000 == 0));
+
+        if info_log {
+            tracing::info!(?height, "ChainVerifier received block");
+        } else {
+            tracing::debug!(?height, "ChainVerifier received block");
+        }
+
+        // Log a warning on unexpected high blocks
+        let is_unexpected_high_block = match height {
+            Some(BlockHeight(height))
+                if (height > self.last_block_height.0 + MAX_EXPECTED_BLOCK_GAP) =>
+            {
+                true
+            }
+            Some(height) => {
+                // Update the last height if the block height was expected
+                self.last_block_height = height;
+                false
+            }
+            _ => false,
+        };
```

Comment:
Per https://github.com/ZcashFoundation/zebra/pull/699#issuecomment-662593415 , I think the cause of the issue is that we're feeding wrong data to the verifier, so we should fix it where we're feeding wrong data, rather than having this check here.

Reply:
I agree:
```
                    // Temporary trace, for identifying early high blocks.
                    // We think the downloader or sync service should reject these blocks
```
https://github.com/ZcashFoundation/zebra/commit/4d742b64db0fa04a218d15580edfa4ddbaa00c04#diff-9ce8cedb3963941e6cffe1a307d9dafcR137-R138